src/solitaire.jl

```diff
@@ -1,4 +1,4 @@
-include("common.jl")
+# include("common.jl")
```

Comment:
Oops, I missed that: it's there because it defines letters_only, rotateRight and so on. Currently slightly baffled about why the tests passed.